vcf-dumper/vcf-dumper-lib/src/test/java/uk/ac/ebi/eva/vcfdump/VariantExporterControllerTest.java

```diff
@@ -204,9 +170,23 @@ public void testVcfExportOneStudy()
         String outputFile = controller.getOuputFilePath();
         testOutputFiles.add(outputFile);
         assertEquals(0, controller.getFailedVariants());   // test file should not have failed variants
-        QueryOptions query = getQuery(studies);
-        VariantDBIterator iterator = variantDBAdaptor.iterator(query);
-        assertEqualLinesFilesAndDB(outputFile, iterator);
+
+        List<VariantMongo> variants = variantRepository.findAll();
```

Comment:
Extract this block to a function, because the code is repeated:
```
long variantCountInDb = countStudiesVariantsInDb(studies);
assert(variantCountInDb != 0);
....
```